bin/citgm-all.js

```diff
@@ -17,6 +20,16 @@ yargs = commonArgs(yargs)
     alias: 'f',
     type: 'boolean',
     description: 'Ignore flaky flags. Don\'t ignore any failures.'
+  })
+  .option('parallel', {
+    alias: 'j',
+    type: 'number',
+    description: 'Number of tests to run in parallel'
+  })
+  .option('autoParallel', {
```

Comment:
Nit: This option may be overkill. Maybe it can be done `--parallel=auto` or simply `--parallel` or something?